src/app/pages/organization/companies/company/company.component.html

```diff
@@ -19,12 +19,15 @@
                   </div>
                   <div class="back back-background"
                     [ngStyle]="{'background-image': 'url(' + logo + ')', 'background-size' : 'contain', 'background-repeat': 'no-repeat'}">
-                    <div *ngIf="this.isAdmin" class="card-body">
+                    <div
+                      *ngIf="this.canCreateCompany && this.canReadCompany && this.canUpdateCompany && this.canDeleteCompany"
```

Comment:
I understand the isAdmin now, only 1 var instead of 4 :-)
Maybe you can add a new member called canCrudCompany which is init with 'this.canCreateCompany && this.canReadCompany && this.canUpdateCompany && this.canDeleteCompany'.
No need to have 'this' in the template, it works without